consensus/istanbul/backend/backend.go

```diff
@@ -468,15 +471,24 @@ func (sb *Backend) Verify(proposal istanbul.Proposal) (time.Duration, error) {
 	if txnHash != block.Header().TxHash {
 		return 0, errMismatchTxhashes
 	}
-
-	// The author should be the first person to propose the block to ensure that randomness matches up.
-	addr, err := sb.Author(block.Header())
+	minVersion := &params.VersionInfo{Major: 1, Minor: 1, Patch: 0}
+	version, err := blockchain_parameters.GetMinimumVersion(nil, nil)
```

Comment:
I expect this will be a pattern we use frequently, so consider adding something that allows you to do:
```
if blockchain_parameters.IsMinimumVersionAtLeast(1, 1, 0) {
  addr, err := sb.Author(block.header())
}
```